client/templates/includes/gillespyResultsEnsemble.pug

```diff
@@ -18,19 +18,19 @@ div#workflow-results-ensemble.card.card-body
 
         tbody
           tr
-            td: div(data-hook="title")
-            td: div(data-hook="xaxis")
-            td: div(data-hook="yaxis")
+            td: div(id="title", data-hook="title")
+            td: div(id="xaxis", data-hook="xaxis")
+            td: div(id="yaxis", data-hook="yaxis")
 
     div.card.card-body
       div
 
         h5.inline Plot Standard Deviation Range
-        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-stddevrange", data-hook="collapse-stddevrange") -
+        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-stddevrange", id="collapse-stddevrange", data-hook="collapse-stddevrange") -
```

Comment:
The element on line 31 already has this id value I would suggest adding '-btn' to the end of the existing id value.

No element use my suggested id value